tensorboard/plugins/histogram/histograms_demo.py

```diff
@@ -29,40 +31,42 @@ def run_all(logdir, verbose=False):
   """Generate a bunch of histogram data, and write it to logdir."""
   del verbose
 
+  tf.set_random_seed(0)
+
   k = tf.placeholder(tf.float32)
 
   # Make a normal distribution, with a shifting mean
   mean_moving_normal = tf.random_normal(shape=[1000], mean=(5*k), stddev=1)
   # Record that distribution into a histogram summary
-  tf.summary.histogram("normal/moving_mean", mean_moving_normal)
+  histogram_summary.op("normal/moving_mean", mean_moving_normal)
```

Comment:
Ah, I see. @dandelionmane @chihuahua @jart what do you think? `tensorboard.plugins.histogram.histogram_op` vs. `tensorboard.plugins.histogram.op`?